app/api/v1/entities/full.rb

```diff
@@ -27,6 +27,20 @@ class IdentityFull < IdentityShallow
             :last_name,
             :ldap_uid
   end
+  
+  class HumanSubjectsInfoFull < HumanSubjectsInfoShallow
+    root 'human_subjects_info', 'human_subjects_info'
```

Comment:
I believe the first root param should be pluralized. https://github.com/intridea/grape-entity#collection-exposure